examples/config-policy.json

```diff
@@ -4,107 +4,63 @@
     "rootDirectory": "/tmp/zot"
   },
   "http": {
-    "address": "127.0.0.1",
-    "port": "8080",
-    "realm": "zot",
     "auth": {
       "htpasswd": {
         "path": "test/data/htpasswd"
-      },
-      "failDelay": 1
+      }
     },
     "accessControl": {
-      "**": {
-        "anonymousPolicy": ["read"],
-        "policies": [
-          {
-            "users": [
-              "charlie"
-            ],
-            "actions": [
-              "read",
-              "create",
-              "update"
-            ]
-          }
-        ],
-        "defaultPolicy": [
-          "read",
-          "create",
-          "delete",
-          "detectManifestCollision"
-        ]
-      },
-      "tmp/**": {
-        "defaultPolicy": [
-          "read",
-          "create",
-          "update"
-        ]
-      },
-      "infra/**": {
-        "policies": [
-          {
-            "users": [
-              "alice",
-              "bob"
-            ],
-            "actions": [
-              "create",
-              "read",
-              "update",
-              "delete"
-            ]
-          },
-          {
-            "users": [
-              "mallory"
-            ],
-            "actions": [
-              "create",
-              "read"
-            ]
-          }
-        ],
-        "defaultPolicy": [
-          "read"
-        ]
+      "groups": {
+        "group1": {
+          "users": ["jack", "john", "jane", "ana"]
+        },
+        "group2": {
+          "users": ["alice", "mike", "jim"]
+        }
       },
-      "repos2/repo": {
-        "policies": [
-          {
-            "users": [
-              "charlie"
-            ],
-            "actions": [
-              "read",
-              "create"
-            ]
-          },
-          {
-            "users": [
-              "mallory"
-            ],
-            "actions": [
-              "create",
-              "read"
-            ]
-          }
-        ],
-        "defaultPolicy": [
-          "read"
-        ]
+      "repos": {
+        "**": {
+          "policies": [{
+            "users": ["charlie"],
+            "groups": ["admins","developers"],

```

Comment:
The groups used in the `policies` are not defined under `groups`. Let's give an example of a valid configuration.
Is there a specific reason for them not to match?

Reply:
@aokirisaki are the group membershipt taken from ldap?
Or is it hardcoded in the config file?